ipmof/energymap.py

```diff
@@ -29,6 +29,11 @@ def energy_map(sim_par, mof, atom_list, export=True, export_dir=sim_dir['energy_
     Resulting energy map is structured as follows:
         emap[0] = [x, y, z, atom1_energy, atom2_energy, atom3_energy, ...]
     """
+    # Initialize MOF and extend structure for energy map calculation
+    mof = MOF(mof_path)
+    mof.force_field(force_field)
```

Comment:
Consider using a property so you can just write something like
mof.force_field = force_field 
and it will be clearer what it means

cons

http://proquest.safaribooksonline.com/book/programming/python/9780134034416/4dot-metaclasses-and-attributes/ch04lev1sec2_html